### DISCUSSION TITLE
Countries / Cities / Places Tracker

### DISCUSSION THREAD

**Discussion by hanwg** [IgnisDa/ryot#833] [Ideas]
Just a suggestion a travel tracker:
- date/month/year of visit
- a short description e.g. was it a business trip? who did i travel with?
- links to photo albums
- a review or summary section to pen down thoughts about the trip
- possibly a map/timeline

**Comment by IgnisDa** [comment]
I think this would be solved by #73?

**Reply by IgnisDa** [reply]
Not sure about the map/timeline thing. Will have to think something for that.

**Reply by StoffelCPR** [reply]
An entire map is quite difficult. And there's a saying "you can do anything, but you can't do everything" I like the idea of Tracking Trips and/or Places to have visited (like a Travel Wishlist) but would ultimately opt to host my GPS data on an app like Dawarich (https://github.com/Freika/dawarich) and a FOSS GPS software on my phone/Google timeline exports to host all GPS data on that app and merely have it integrate with RYOT in some way (RYOT as the "FrontEnd" to deeper more complex or specialized paltforms seems an incredible feature)

As for the photo part -> Take an app like immich and self-host your Google Photos alternative and then have it integrate with Ryot in some way

**Reply by IgnisDa** [reply]
hmm yeah i agree with an integration aspect too.
